### PR TITLE
Bl tabbed interface

### PR DESCRIPTION
**Pull request by tluu5**

### CODE REVIEW

File: app/views/users/liked.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div class="col md-6 offset-3">
```
Comment (armstrca) [inline app/views/users/liked.html.erb]: change to `col-md-6 offset-3`

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<h1>
+  <%= @user.username%>
+</h1>
+
+<h2>Own photos</h2>
+
+<div class="row">
+  <div class="col md-6 offset-3">
```
Comment (armstrca) [inline app/views/users/show.html.erb]: change to col-md-6 offset-3

**Review by armstrca — CHANGES_REQUESTED** [review]

File: app/views/photos/_photo.html.erb [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
-    <%= photo.caption %>
-  </p>
-
-  <p>
-    <strong>Owner:</strong>
-    <%= photo.owner_id %>
-  </p>
-
+<div class="row mb-4">
+  <div class="col-md-6 offset-md-3">
```
Comment (jessjno) [inline app/views/photos/_photo.html.erb:2]: good job fixing this one

File: app/controllers/comments_controller.rb
Comment (jessjno) [inline app/controllers/comments_controller.rb:1]: good job changing the redirect_to

File: app/views/users/show.html.erb (hunk 2 above)
Comment (jessjno) [inline app/views/users/show.html.erb]: don't forget to add that first '-' in 'col-md-6'

**Review by jessjno — APPROVED** [review]
you should run and search and replace to make sure you got all the 'col-md-6' ones correct

File: config/routes.rb
Comment (armstrca) [inline config/routes.rb:1]: Lines 11 through 13 are incomplete and result in errors like this:

```
There's a problem with your routes.rb file.

Missing :controller key on routes definition, please check your routes.

/workspaces/photogram-industrial/config/routes.rb:11
```

**Review by armstrca — CHANGES_REQUESTED** [review]

**Review by armstrca — APPROVED** [review]
Thanks for fixing routes.rb. Looks good.

**Pull request merged**